cogs/misc.py

```diff
@@ -146,12 +142,11 @@ async def events(self, ctx: BoboContext) -> str:
             - float(await self.bot.redis.get('events_start_time'))
         ) / 60
 
-        return dedent(
-            f"""
+        return dedent(f"""
             Total WS Events: {events_count}
             Average WS Events per minute: {events_count // time_difference}
-        """
-        )
+            Alvin S' Will: {ctx}
```

Comment:
Must thy remove this line for this PR to be merged

Reply:
Alvin S, such will only occur if thy revela thy Chinese name